packages/-ember-data/tests/integration/record-data/store-wrapper-test.ts

```diff
@@ -284,7 +284,7 @@ module('integration/store-wrapper - RecordData StoreWrapper tests', function(hoo
 
     class RecordDataForTest extends TestRecordData {
       id: string;
-      _isNew: boolean = false;
```

Comment:
this looks like an accidental removal

Reply:
This is one of the rules, i don't remember which one, but I plan on adding details about each of these changes to the PR.

I think it was something about this being unnecessary

Reply:
ah, it can't be inferred though :( it makes the type `false` instead of `boolean` which then errors when you set it to true

Reply:
These changes were removed